app/Views/ScBooking/BookingType.cshtml

```diff
@@ -111,9 +111,8 @@
 
                     <div id="different-place-of-trial" class="form-group btn-radio-group mb-5">
                         <h6 class="form-section-label">
-                            Do you have an entered court order changing the place of trial or is a different place of
-                            trial stated on the initiating documents (e.g. Notice of Civil Claim, Notice of Family
-                            Claim)?
+                            Do you have an entered court order changing the place of trial, or is this place of trial
+                            stated on the initiating documents (e.g. Notice of Civil Claim, Notice of Family Claim)?
```

Comment:
I wonder if they realized when they were writing this sentence that the form is progressively revealed?

e.g. "...is **this** place of trial..." <= what does "this" refer to?

Reply:
Updating here so I don't forget: Dom passed the ticket back to Winnie to see if we can clarify the wording a bit. That's what I expected so I've held off merging until the wording is revised 👍